test/e2e/standard/osbachecker.go

```diff
@@ -0,0 +1,86 @@
+package standard
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+
+	"github.com/openshift/openshift-azure/pkg/util/random"
+)
+
+type osbaObj struct {
+	name string
+	data []byte
+}
+
+var osbaApp, osbaService osbaObj
+
+func init() {
+	osbaApp.name = "osba-app"
+	osbaService.name = "osba-service"
+
+	var err error
+	osbaService.data, err = ioutil.ReadFile("../manifests/osba/osba-os-template.yaml") // running via `go test`
+	if os.IsNotExist(err) {
+		osbaService.data, err = ioutil.ReadFile("test/e2e/manifests/osba/osba-os-template.yaml") // running via compiled test binary
+	}
+	if err != nil {
+		panic(err)
+	}
+}
+
+// ValidateOSBAApp verifies that the created application runs normally
+func (sc *SanityChecker) ValidateOSBAApp(ctx context.Context, cookie interface{}) []*TestError {
```

Comment:
Do not extend SanityChecker for this.  Put all these functions in or next to test/e2e/specs/osba_app.go.